app/views/home/index.html.erb

```diff
@@ -3,12 +3,18 @@
     <em>The <a href="/newest">newest</a> stories that have not yet reached the front page.</em>
   </div>
 <% end %>
-<% if @tag %>
+<% if @tags.present? %>
   <div class="box" id="leader">
     Stories <a href="/tags">tagged</a> as
-    <%= tag_link(@tag) %>
-    &ndash;
-    <%= @tag.description %>
+    <% @tags.each do |tag| %>
+      <%= tag_link(tag) %>
+
+      <% if tag.description.present? %>
+        &ndash; <%= tag.description %>
+      <% end %>
+    <% end %>
+
+    <%= render(partial: 'tags/multi_tag_tip', locals: { tags: @tags + Tag.limit(1) }) if @tags.count == 1 %>
```

Comment:
I also have a static version of this waiting in the wings - basically it just links to `lobste.rs/t/programming,audio`. I picked out a Tag here because I thought sister sites might not have the same tags, but it does somewhat irritate me to be doing this query.

Reply:
> Tip: read stories across multiple tags with `/t/tag1,tag2`.

Reply:
Done!